examples/README.md

```diff
@@ -29,3 +29,10 @@ These scripts are `trigger_cd_to_deploy_[node|java]_app.sh` - see individual scr
 The scripts will deploy to staging then deploy straight to production (assuming a successful deploy to staging) without running any tests.
 
 There is an additional script to run tests in staging without deploying any applications: `trigger_cd_to_run_tests.sh`
+
+## Updating a variable service
+To update a value in a variable service, all values in that service must be provided. 
+E.g. `cf update-user-provided-service os-places-variable-service -p '{\"OS_PLACES_API_KEY\": \"${OS_PLACES_API_KEY}\", \"OS_PLACES_URI\": \"${OS_PLACES_URI}\" }'`
+
+Any service that uses that variable service needs to be restaged
```

Comment:
Can we add that this will result in some downtime, and that if downtime is to be avoided then they should use the script to trigger a cd build of the relevant app?